src/main/java/org/inferred/freebuilder/processor/Analyser.java

```diff
@@ -70,6 +57,19 @@
 import javax.lang.model.util.SimpleTypeVisitor6;
 import javax.lang.model.util.Types;
 
+import org.inferred.freebuilder.processor.Metadata.Property;
```

Comment:
Why reorder the imports? Let me know if I should be using anything other than the [Google import order](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s3.3.3-import-ordering-and-spacing) :)